apps/wappush/js/cp_screen_helper.js

```diff
@@ -192,7 +188,8 @@ var CpScreenHelper = (function() {
     // Multimedia Messaging System Enabler). In this case we must not continue.
     if (apns.length === 0) {
       message = finishConfirmDialog.querySelector('strong');
```

Comment:
I know you didn't write this, but I find it quite misleading that the same `message` variable is reused here for something that is completely different ("message" object vs DOM node).